tests/utils/command_line_params.cpp

```diff
@@ -498,6 +518,21 @@ std::shared_ptr<QUDAApp> make_app(std::string app_description, std::string app_n
     ->add_option("--precon-schwarz-type", precon_schwarz_type,
                  "The type of Schwarz preconditioning to use (default=invalid)")
     ->transform(CLI::QUDACheckedTransformer(schwarz_type_map));
+
```

Comment:
Move these `MADWF` parameters to a separatae CLI group.

Reply:
https://github.com/lattice/quda/pull/1061/commits/5d9e47921fe9b8a5bcb89857ef843062baaba806